### PR TITLE
Add missing domains

### PR DESCRIPTION
**Pull request by LouiseBonnard**

### CODE REVIEW

File: prestafraud.php [hunk 1]
```diff
@@ -49,39 +49,39 @@ public function __construct()
...
-            20 => $this->l('Travel'),
+            0 => $this->trans('-- Please choose your main activity --', array(), 'Modules.Prestafraud.Admin'),
+            1 => $this->trans('Adult', array(), 'Admin.Global'),
+            2 => $this->trans('Animals and Pets', array(), 'Admin.Global'),
+            3 => $this->trans('Art and Culture', array(), 'Admin.Global'),
+            4 => $this->trans('Babies', array(), 'Admin.Global'),
+            5 => $this->trans('Beauty and Personal Care', array(), 'Admin.Global'),
+            6 => $this->trans('Cars', array(), 'Admin.Global'),
+            7 => $this->trans('Computer Hardware and Software', array(), 'Admin.Global'),
+            8 => $this->trans('Download', array(), 'Admin.Global'),
```
Comment (LouiseBonnard) [inline prestafraud.php]: Is it about virtual products only?
🤔

File: prestafraud.php [hunk 2]
```diff
@@ -49,39 +49,39 @@ public function __construct()
...
+            9 => $this->trans('Fashion and accessories', array(), 'Admin.Global'),
```
Comment (LouiseBonnard) [inline prestafraud.php]:
```suggestion
            9 => $this->trans('Fashion and Accessories', array(), 'Admin.Global'),
```

File: prestafraud.php [hunk 3]
```diff
@@ -49,39 +49,39 @@ public function __construct()
...
+            10 => $this->trans('Flowers, Gifts and Crafts', array(), 'Admin.Global'),
+            11 => $this->trans('Food and beverage', array(), 'Admin.Global'),
```
Comment (LouiseBonnard) [inline prestafraud.php]:
```suggestion
            11 => $this->trans('Food and Beverage', array(), 'Admin.Global'),
```

File: prestafraud.php [hunk 4]
```diff
@@ -49,39 +49,39 @@ public function __construct()
...
+            12 => $this->trans('HiFi, Photo and Video', array(), 'Admin.Global'),
+            13 => $this->trans('Home and Garden', array(), 'Admin.Global'),
+            14 => $this->trans('Home Appliances', array(), 'Admin.Global'),
+            15 => $this->trans('Jewelry', array(), 'Admin.Global'),
+            16 => $this->trans('Mobile and Telecom', array(), 'Admin.Global'),
+            17 => $this->trans('Services', array(), 'Admin.Global'),
+            18 => $this->trans('Shoes and accessories', array(), 'Admin.Global'),
```
Comment (LouiseBonnard) [inline prestafraud.php]:
```suggestion
            18 => $this->trans('Shoes and Accessories', array(), 'Admin.Global'),
```

File: prestafraud.php [hunk 5]
```diff
@@ -49,39 +49,39 @@ public function __construct()
...
         $this->_payment_types = array(
-            0 => $this->l('Cheque'),
-            1 => $this->l('Bankwire'),
-            2 => $this->l('Credit card'),
-            3 => $this->l('Credit card multiple'),
-            4 => $this->l('Prepaid account (MoneyBookers, PayPal...)'),
+            0 => $this->trans('Check', array(), 'Admin.Global'),
+            1 => $this->trans('Bank wire', array(), 'Admin.Global'),
+            2 => $this->trans('Credit card', array(), 'Admin.Global'),
+            3 => $this->trans('Credit card multiple', array(), 'Admin.Global'),
```
Comment (LouiseBonnard) [inline prestafraud.php:83]: What could it be?

**Comment by matks** [comment]
@LouiseBonnard this module has not been modified since 2017, and before 2015 🤔 and it does not look like a native module

Do you know if it's still used ? Maybe we should just drop it ?

**Comment by LouiseBonnard** [comment]
@matks, it is still available in the back office (in Modules > Modules Catalog), it is called PrestaShop Security. ;-)

**Comment by matks** [comment]
It's not because it's available that it's maintained. I think any software that has not been modified for 4 years can be considered obsolete.

**Comment by LouiseBonnard** [comment]
So we're good!

**Comment by PierreRambaud** [comment]
@LouiseBonnard You need to update the code with your suggestions maybe? :wink:

**Comment by LouiseBonnard** [comment]
I'd like someone else's feedback before, there are wordings that seem outdated to me... @sarahdib perhaps?

File: prestafraud.php (hunk 1 above)
Comment (sarahdib) [reply prestafraud.php]: No it's about all type of product

File: prestafraud.php (hunk 5 above)
Comment (sarahdib) [reply prestafraud.php:83]: This part of the module associates payment types ex: 
a CB module is installed on the shop, it associated to credit card

Comment (LouiseBonnard) [reply prestafraud.php:83]: Hello @sarahdib, thanks for your answers! I got this part about all the payment methods but what does "credit card multiple" refer to?
🤔

File: prestafraud.php (hunk 1 above)
Comment (LouiseBonnard) [reply prestafraud.php]: All downloadable products? It makes no sense to me to have a whole list of types and, in the end, an entry called "Download". If it regards virtual products (downloadable products), I think the best would be to call it "Virtual Products". :-)

Comment (sarahdib) [reply prestafraud.php]: @LouiseBonnard ok I see yes "Donwload" in this contexte it's a virtual prouct

File: prestafraud.php (hunk 5 above)
Comment (sarahdib) [reply prestafraud.php:83]: @LouiseBonnard some module allows payment 2x 3x ... this is the credit card multiple

File: prestafraud.php (hunk 1 above)
Comment (LouiseBonnard) [reply prestafraud.php]:
```suggestion
            8 => $this->trans('Virtual Products', array(), 'Admin.Global'),
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by colinegin** [comment]
This PR can be merged !

**Comment by PierreRambaud** [comment]
Thanks @LouiseBonnard

**Pull request merged**